pkg/inputs/flow/format.go

```diff
@@ -106,6 +111,12 @@ func (t *KentikDriver) toJCHF(fmsg *flowmessage.FlowMessage) *kt.JCHF {
 		dev.SetUserTags(in.CustomStr)
 	} else {
 		in.DeviceName = net.IP(fmsg.SamplerAddress).String()
+		if t.resolv != nil {
+			dm := t.resolv.Resolve(t.ctx, in.DeviceName)
+			if dm != "" {
+				in.DeviceName = dm
+			}
+		}
```

Comment:
@i3149 - can we add a one-time `Info` level log message that shows the results of this check?

```
$src_addr resolved to $device_name
```

something along those lines? just so we have tracking on the attempt.

Reply:
definitely.